revolv/static/dashboard.scss

```diff
@@ -1,5 +1,23 @@
 @import "revolv-settings";
 
+@import "utils";
+@import "foundation-settings";
+@import "bower_components/foundation/scss/foundation";
+@import "revolv-settings";
+
+
+#project-menu {
+    background-color: orange;
+}
+
+$divider-font-size: 14px;
+.homepage-divider {
+    @include text-with-divider($divider-font-size, #ddd);
+    color: $revolv-blue;
+    height: 0.6em; // adjustment for helvetica neue
```

Comment:
This is also on the homepage right now so we should extract it into a mixin yes?